README.md

```diff
@@ -1515,6 +1515,265 @@ __NOTE__: This section does not go over any additional tuning that can be done
 for the kernel datapath.
 
 
+## Test with Open vSwitch full offload (data and control path) to NIC
```

Comment:
Can you please move this to the bottom of the readme. 

Secondly, can you change the full offload to something different, as this term is already used for other kind of offload. Maybe call it something like "Test with Open vSwitch running on the NIC"